src/providers/setting-service.ts

```diff
@@ -73,7 +73,7 @@ export class SettingService {
   }
 
   isShowRestUrlField(): Observable<boolean> {
-    return this.readKey(this.isShowRestUrlFieldKey).map(val => val == null ? this.isShowRestUrlFieldDefault : val);
+    return this.readKey(this.isShowRestUrlFieldKey).map(val => val === undefined ? this.isShowRestUrlFieldDefault : val);
```

Comment:
I think mapping null to undefined in `readKey()` would mask a limitations of the underlying API, as `Storage.get()` returns null by design when the key does not exists:
https://github.com/ionic-team/ionic-storage/issues/7
https://github.com/localForage/localForage/issues/667
https://localforage.github.io/localForage/#data-api-getitem